Xappy/Xappy/Content/About/IndexPage.xaml

```diff
@@ -5,8 +5,7 @@
              xmlns:global="clr-namespace:Xappy.Domain.Global"             
              x:Class="Xappy.About.IndexPage"
              Title="About"
-             Shell.TabBarIsVisible="false"
-             BackgroundColor="{StaticResource ColorAppBackground}">
+             Shell.TabBarIsVisible="false">
```

Comment:
I've added this that is Theme aware. `BackgroundColor="{DynamicResource backgroundColor}"` Use that instead?